### PR TITLE
Embarc 308/show delete if timed out

### PR DESCRIPTION
**Pull request by KristenWAlthoff**
- Add the delete button in the case that Connect times out so users still have the ability to delete user and start the flow over again

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,35 +6,40 @@ This example application will load a React web app where you can enter utility c
 
 You'll need your Arcadia API keys, a computer that can run Node.js version 14.x, and familiarity with the command line.
 
-
 ## Configuration
 
 The first step is to clone this example application. Use `git` to download the repository:
+
```
Comment (KristenWAlthoff) [inline README.md:12]: I think this was just my linter added on autosave but can revert if needed

File: src/connect-widget.jsx [hunk 2]
```diff
@@ -85,32 +92,53 @@ const ConnectWidget = () => {
...
-    )
+      <div>
+        <p>Uh oh! We hit a problem: {error} </p>
+        {renderConnectBtn}
+      </div>
+    );
   }
 
   if (timedOut) {
-    return <p>Utility Credential #{utilityCredentialId} was created but the credentials weren't verified before the Component timed out and closed. The credentials will be verified in the background. If you've configured a webhook, check your console for incoming webhooks about verification.</p>
```
Comment (KristenWAlthoff) [inline src/connect-widget.jsx:96]: This is the part that actually matters. The other changes are from the linter.

**Review by collinclarke — COMMENTED** [review]
The important stuff looks good. There are two things I would revert:
1. The package-lock.json change (unless there was a good reason for it to update)
2. Deletion of .env-example

Not sure why your linter is doing so much here since I don't think this project has a lint file, but none of the changes it made seem like an issue so 👍

**Comment by KristenWAlthoff** [comment]
> Thanks, other files are now reverted! Hmm, maybe it's my VSCode autosave settings? Will investigate.

**Review by BrandonGoren — APPROVED** [review]
Looks good to me

**Pull request merged**